R/appshot.R

```diff
@@ -40,14 +42,73 @@ appshot.character <- function(app, file = "webshot.png", ...,
     p <- processx::process$new("R", args = c("--slave", "-e", cmd))
```

Comment:
Switch to `callr::r_bg`. We might not even need the `processx` package as a (direct) dependency.